db/schema.rb

```diff
@@ -513,6 +513,8 @@
     t.boolean "open_membership", default: false, null: false
     t.boolean "accepting_observations", default: true, null: false
```

Comment:
@mo-nathan:
I removed all code references to `accepting_observations`.
But I left to column here for now in order to facilitate switching branches.
I plan to remove the `accepting_observations` column after this PR merged.

Reply:
Makes sense.